### DISCUSSION TITLE
0.10.6

### DISCUSSION THREAD

**Discussion by danwalmsley** [AvaloniaUI/Avalonia#6015] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/AvaloniaUI/Avalonia/releases/tag/0.10.6'>0.10.6</a>.</em>

**Comment by fforjan** [comment]
@danwalmsley I really like what you did for the 0.10.5 with the list of new features & bugfixes.
Is it something you could continue to do for newer release ?

**Reply by maxkatz6** [reply]
@fforjan done

**Reply by fforjan** [reply]
And to be really picky, is it possible to ensure you add info when it is platform specific, although it seems obvious as example fo this one, 
_Added support for single-touch screens directly via evdev #5894_ 
would be better as :
_[Linux] Added support for single-touch screens directly via evdev #5894_

**Comment by workgroupengineering** [comment]
Hi, why not create more milestones? With milestones it is easier to understand how to contribute for those who are not members of the core team.
